reduction/lr_reduction/template.py

```diff
@@ -225,7 +222,7 @@ def process_from_template_ws(ws_sc, template_data, q_summing=False,
                                                    theta=np.abs(theta),
                                                    dead_time=template_data.dead_time,
                                                    paralyzable=template_data.paralyzable,
-                                                   functional_background=functional_background,
```

Comment:
This allow for the erroneous use-case  `peak_bck` having four elements (two backgrounds) and `functional_background=False`. Are we leaving the responsibility to the user to set `functional_background=True` when `peak_bck` has four elements?

Reply:
The intent is that `functional_background=False` only uses the first two bck entries. The `peak_bck` has always had four elements anyway (padded with zeros at the end), so the number of entries is not meant as an indicator. There is a check in the code to catch `function_background=True` but only two entries are given. So we should be good.